config/default.yml

```diff
@@ -3826,9 +3826,10 @@ Style/TrailingCommaInArrayLiteral:
   StyleGuide: '#no-trailing-array-commas'
   Enabled: true
   VersionAdded: '0.53'
+  # If `comma`, the cop requires a comma after the last item in an array,
   # but only when each item is on its own line.
   # If `consistent_comma`, the cop requires a comma after the last item of all
-  # non-empty array literals.
+  # non-empty, multiline array literals.
```

Comment:
Are you sure about this? I thought this was enforced for single-line literals as well?

Reply:
Yes,
if we check https://github.com/rubocop-hq/rubocop/blob/master/lib/rubocop/cop/mixin/trailing_comma.rb#L70 style is enforced only in case of multiline